remo/settings/base.py

```diff
@@ -300,10 +298,6 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Paginator settings
 ITEMS_PER_PAGE = 20
 
-# django-celery setup
-import djcelery  # noqa
-djcelery.setup_loader()
```

Comment:
Shouldn't all the `djcelery` related entries be part of the update to `3.1.25` version?